### PR TITLE
Stop generating precomputed_values.ml

### PR DESCRIPTION
**Pull request by martyall**
current `gen_values.exe` benchmark: `356.00566864013672ms`

### CODE REVIEW

**Comment by martyall** [comment]
!ci-build-me

File: src/lib/precomputed_values/precomputed_values.ml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    ; proof_level = Genesis_constants.Proof_level.for_unit_tests
+    ; genesis_constants = Genesis_constants.for_unit_tests
+    ; genesis_ledger = Genesis_ledger.for_unit_tests
+    ; genesis_epoch_data = Consensus.Genesis_epoch_data.for_unit_tests
+    ; genesis_body_reference
+    ; consensus_constants = Lazy.force Consensus.Constants.for_unit_tests
+    ; protocol_state_with_hashes
+    ; constraint_system_digests =
+        lazy
+          (List.map (Lazy.force hashes) ~f:(fun (x, y) ->
```
Comment (mrmr1993) [inline src/lib/precomputed_values/precomputed_values.ml:46]: This `List.map` is now not useful, you can just replace the whole expressions with `hashes`.

Comment (martyall) [reply src/lib/precomputed_values/precomputed_values.ml:46]: should have cranked that reduction machine a little harder :grimacing:

**Review by mrmr1993 — APPROVED** [review]

**Comment by martyall** [comment]
!ci-build-me

**Pull request merged**